workbench-core/datasets/src/schemas/createDataSet.ts

```diff
@@ -13,10 +13,11 @@ const CreateDataSetSchema: Schema = {
     datasetName: { type: 'string' },
     storageName: { type: 'string' },
     path: { type: 'string' },
-    awsAccountId: { type: 'string' }
+    awsAccountId: { type: 'string' },
+    region: { type: 'string' }
```

Comment:
Will that validate that empty string is not allowed (and should it be allowed)?

Reply:
Empty string is and should be allowed. This schema is to validate the request body structure so no additional parameters and parameters of an incorrect type cannot be passed in.